doc/api/report.md

```diff
@@ -519,6 +519,10 @@ includes the date, time, PID and a sequence number. The sequence number helps
 in associating the report dump with the runtime state if generated multiple
 times for the same Node.js process.
 
+Diagnostic report has an associated single-digit version number (`report.header.reportVersion`),
+uniquely representing the report format. The version number is bumped
+when new key is added or removed, or the data type of a value is changed.
```

Comment:
Does this mean that it can be possible to bump the version more than once in a single Node.js version release?

Reply:
It should be limited to one bump per Node version. Otherwise there will be versions that you can only access via building from `main`.

Reply:
I believe it would be worth to document that the report versions (with their new fields/changes) would keep aligned across LTS releases. For example, report version 9-to-be-released will be published in all LTS versions with the identical set of changes.

Reply:
> I believe it would be worth to document that the report versions (with their new fields/changes) would keep aligned across LTS releases.

@legendecas - thanks, pushed a line to that effect, PTAL.

In terms of bumping per change versus per release: I favour the former because:
 - consistency: one unique version mapping to one set of changes.
 - we might be already doing similar things with upstream components such as v8
 - there are users who build from source too, who will benefit from this model.

what do you think?